crawler/alexa.js

```diff
@@ -0,0 +1,24 @@
+import { default as JSZip } from 'jszip';
+
+function getTop1M(zipURL) {
+  if (!zipURL) {
+    zipURL = 'http://s3.amazonaws.com/alexa-static/top-1m.csv.zip';
+  }
+  const request = new Request(zipURL);
+  return fetch(request)
+  .then(response => response.blob())
+  .then(file => {
+    const zip = new JSZip(file);
+    return zip.file('top1m.csv').asText();
+  });
+}
+
+function parseTop1M(csv) {
+  const content = csv.split('\n');
+  return content.map(row => row.split(',')[1]);
+}
+
+export default {
+  getTop1M,
+  parseTop1M,
+};
```

Comment:
This file looks great! Clean and effective.

To make this fit with the existing architecture, please move this file to crawler/urlJobPopulatorBackends/alexa.js and make it expose the API that is expected by crawler/urlJobPopulator.js